src/Application.vala

```diff
@@ -47,6 +46,29 @@ public class Music.Application : Gtk.Application {
         ((SimpleAction) lookup_action (ACTION_PREVIOUS)).set_enabled (false);
         ((SimpleAction) lookup_action (ACTION_SHUFFLE)).set_enabled (false);
 
+        var granite_settings = Granite.Settings.get_default ();
+        var gtk_settings = Gtk.Settings.get_default ();
+
+        gtk_settings.gtk_icon_theme_name = "elementary";
+        gtk_settings.gtk_theme_name = "io.elementary.stylesheet.orange";
+
+        gtk_settings.gtk_application_prefer_dark_theme = (
+            granite_settings.prefers_color_scheme == DARK
+        );
+
+        granite_settings.notify["prefers-color-scheme"].connect (() => {
+            gtk_settings.gtk_application_prefer_dark_theme = (
+                granite_settings.prefers_color_scheme == DARK
+            );
+        });
+    }
+
+    protected override void activate () {
+        if (active_window != null) {
+            active_window.present_with_time (Gdk.CURRENT_TIME);
```

Comment:
```suggestion
            active_window.present ();
```

present_with_time() is never the right choice when there isn't a timestamp at hand.

in this case in particular Gtk.Application already handle the timestamp for us, so present() should work fine.